lib/js-api/src/alfrescoApi.ts

```diff
@@ -69,6 +69,11 @@ export class AlfrescoApi implements Emitter, AlfrescoApiType {
         this.storage = Storage.getInstance();
         this.storage.setDomainPrefix(config.domainPrefix);
 
+        this.initConfig(config);
+        this.validateTicket(config);
+    }
+
+    private initConfig(config: AlfrescoApiConfig) {
```

Comment:
can you please add some sort of unit testing to verify the changes? thanks

Reply:
Added unit test.